Clicker/ViewControllers/AttendanceViewController.swift

```diff
@@ -19,12 +19,14 @@ class AttendanceViewController: UIViewController {
     var navigationTitleView: NavigationTitleView!
     var collectionView: UICollectionView!
     var exportButton: UIButton!
+    var selectAllBarButtonItem: UIBarButtonItem!
 
     // MARK: - Data vars
     weak var delegate: AttendanceViewControllerDelegate?
     var adapter: ListAdapter!
     var session: Session!
     var pollsDateAttendanceArray: [PollsDateAttendanceModel] = []
+    var unSelect: Bool = false
```

Comment:
I feel like this variable name is a bit confusing, can you rename it to `isSelected` or something similar?